src/components/content/stats/stats.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import { theme } from '../../layout/theme';
+import { up } from '../../support/breakpoint';
+import { TextStyles } from '../../typography';
+
+interface StatProps {
+  title: string;
+  content: string;
+}
+
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+
+const Title = styled.div`
+  ${TextStyles.toplineR}
+  color: ${theme.palette.text.topline};
+  margin-bottom: 12px;
+`;
+
+const Content = styled.div`
+  ${TextStyles.headlineM}
+`;
+
+const StatsWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 48px;
+
+  ${up('sm')} {
+    flex-direction: row;
+  }
+`;
```

Comment:
Lets replace those `div` tags with more meaningful ones, like the [dl tag](https://www.w3schools.com/TAGs/tag_dl.asp):

### Example HTML result
```html
<dl>
  <dt>Team Size</dt>
  <dd>6 people</dd>
  <dt>Start</dt>
  <dd>December 2018</dd>
  <dt>End</dt>
  <dd>Ongoing</dd>
</dl>
```

Since we need to remove the `Wrapper` (`StatsItem` wrapper), we should switch to a css grid for the `StatsWrapper`. 

I have created a small codepen as orientation for you: [Codepen Example](https://codepen.io/dschu/pen/LYegRNv)

Reply:
thank you for suggestion, this is convenient, I changed it 👍